lxqt-base/lxqt-session/lxqt-session-0.12.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="LXQT session manager"
+HOMEPAGE="http://lxqt.org/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxde/${PN}.git"
+else
+	SRC_URI="https://github.com/lxde/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="GPL-2 LGPL-2.1+"
+SLOT="0"
+
+RDEPEND="
+	>=dev-libs/libqtxdg-1.0.0:=
+	dev-qt/qtcore:5=
+	dev-qt/qtdbus:5=
+	dev-qt/qtgui:5=
+	dev-qt/qtwidgets:5=
+	dev-qt/qtx11extras:5=
+	dev-qt/qtxml:5=
+	kde-frameworks/kwindowsystem:5=[X]
+	~lxqt-base/liblxqt-${PV}
+	x11-libs/libX11
+	x11-misc/xdg-user-dirs
+	!!lxqt-base/lxqt-common
```

Comment:
I think `~x11-themes/lxqt-themes-${PV}"` should be a dependency here too.

Reply:
Do you propose it to be a hard dependency? Or should USE-flag 'themes' be introduced here too similar to lxqt-meta (or moved from lxqt-meta)?

Reply:
Having it as a use flag is fine for me, would be `themes ? ~x11-themes/lxqt-themes-${PV}`. But it definitely should set by default. Having no theme makes lxqt unusable.

I think someone could have a custom theme and ditch the default theme. That's why the use flag is fine.

Reply:
Also, in their release notes, they suggests only lxqt-session depends on the theme:

See section "Notes For Packagers" here: https://lxqt.org/release/2017/10/21/lxqt-0120/

```
Please also be aware that lxqt-qtplugin needs to pickup a new dependency libfm-qt(-dev). lxqt-session should pick up lxqt-themes or lxqt-theme.
```

Reply:
I'll update it.